the-webauthn-server/the-symfony-way/entities-with-doctrine.md

```diff
@@ -22,36 +22,37 @@ declare(strict_types=1);
 
 namespace App\Entity;
 
-use App\Repository\PublicKeyCredentialSourceRepository;
-use Doctrine\ORM\Mapping\Entity;
-use Doctrine\ORM\Mapping\Column;
-use Doctrine\ORM\Mapping\GeneratedValue;
-use Doctrine\ORM\Mapping\Id;
-use Doctrine\ORM\Mapping\Table;
 use Symfony\Component\Uid\Ulid;
-use Webauthn\PublicKeyCredentialSource as BasePublicKeyCredentialSource;
+use Doctrine\ORM\Mapping as ORM;
```

Comment:
I am not a big fan of this syntax.
In any case and because this is just an example, you are free to change it with your preferred coding standards

Reply:
"standard" of symfony entity when you create a entity from console.